src/components/LiquidityMining/Create/Steps/PreviewAndCreate/index.tsx

```diff
@@ -23,16 +40,55 @@ const ResponsiveContainer = styled(Box)<{ flex1?: boolean }>`
     margin-top: 16px !important;
   `}
 `
+const StyledCampaignCard = styled(CampaignCard)`
+  width: 50%;
+`
+const CampaignDetailWrapper = styled(Flex)`
+  gap: 32px;
+`
+const SimulatedValue = styled.div`
+  font-family: 'Fira Mono';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 22px;
+  /* identical to box height */
+
+  text-align: right;
+  letter-spacing: 0.02em;
+  text-transform: uppercase;
 
+  color: ${props => props.theme.text2};
+`
+const StyledSwitch = styled(Repeat)`
+  width: 12px;
+  height: 12px;
+  stroke: ${props => props.theme.text5};
+  margin-left: 4px;
+`
+const SwitchContainer = styled.div`
+  font-size: 10px;
+  display: flex;
+  font-weight: 600;
+  color: ${props => props.theme.text5};
+  line-height: 11px;
+  letter-spacing: 0.15em;
+  text-transform: uppercase;
+  cursor: pointer;
+`
+const dollarAmountMaxSimulation = 10000000
```

Comment:
Can you declare this with a SCREAMING_SNAKE_CASE please?
```suggestion
const DOLLAR_AMOUNT_MAX_SIMULATION = 10000000
```